lib/ansible.sh

```diff
@@ -287,8 +287,8 @@ strap::ansible::playbook::run() {
     [[ "$1" == --*=* ]] && set -- "${1%%=*}" "${1#*=}" "${@:2}" # normalize `--foo=bar` into `--foo bar`
     case "$1" in
       --playbook|--with-playbook)
-        playbook_dir="${2:-${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default}"
-        [[ -d "${playbook_dir}" ]] || strap::abort "strap lansible: $1 needs to be a directory"
+        playbook_file="${2:-${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml}"
```

Comment:
Out of curiosity - why is this being changed to a file instead of a directory?  What additional benefit does that have?

Reply:
So it came to me that it makes more sense to have an actual playbook file following `--playbook` instead of a playbook dir. Not much of a benefit but just looks more natural to me 😛

Reply:
I think this logic is a little incorrect:  if the user specifies the `--playbook` or `--with-playbook` flags, they *must* specify a playbook directory or file.  We shouldn't default to the one in the working dir when using that flag.

We should only default to the working directory if the flag isn't specified and a playbook is found in the working directory.

Reply:
OK, on this on I agree, since in `cmd/lansible` we alway explicitly use `--playbook`. The original logic did a similar thing but now I can remove this part of defaulting logic.